src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -224,19 +194,11 @@ end
 
 # this will only work for iterative solves
 # direct solvers use prefactorization
-isadjustable(ark::AdditiveRungeKutta) = ark.implicitoperator! isa EulerOperator
 function updatedt!(ark::AdditiveRungeKutta, dt)
-    @assert isadjustable(ark)
+    @assert dtisadjustable(ark.besolver!)
```

Comment:
Can/should we change this function name to `Δt_is_adjustable`?

Reply:
Done.